docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -1,62 +1,295 @@
 # Run MXNet Scala Examples Using the IntelliJ IDE
 
+<!--TODO: switch to new namespace and new maven package -->
+
 This tutorial guides you through setting up a Scala project in the IntelliJ IDE and shows how to use an MXNet package from your application.
 
 ## Prerequisites:
-To use this tutorial, you need:
+To use this tutorial you need the following items, however after this list, installation info for macOS is provided for your benefit:
+
+- [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Maven](https://maven.apache.org/install.html)
+- [Scala](https://www.scala-lang.org/download/) - comes with IntelliJ, so you don't need to install it separately
+- [MXNet Shared Library and Scala Package](#build-the-mxnet-shared-library-and-scala-package)
+- [IntelliJ IDE](https://www.jetbrains.com/idea/)
```

Comment:
Do we really want to have IDEs like IntelliJ as part of the prerequisite?

Reply:
Fixed the conflict...

Well this is purpose of the tutorial... to setup intellij.
The scala install page won't have this requirement... just an option.